### ISSUE TITLE
Add More Information to Restaurant Notes

### ISSUE THREAD

**Issue by jfc3** [jfc3/10kapart#58]
Need to add more Information to restaurant notes filed including meal to order or dishes to get, along with when's the best time to go.

**Comment by jfc3** [comment]
Need to update the following city, state, and location JSON files.

- [ ] Annapolis, MD
- [ ] Austin, TX
- [ ] Baltimore, MD
- [ ] Boston, MA
- [ ] Buffalo, NY
- [ ] Cleveland, PA
- [ ] Columbus, OH
- [ ] Delaware
- [ ] Las Vegas, NV
- [ ] Maryland
- [ ] Memphis, TN
- [ ] Mississippi
- [ ] New Orleans, LA
- [ ] New York, NY
- [ ] Northern Virginia
- [ ] Ocean City, NJ
- [ ] Philadelphia, PA
- [ ] Portland, OR
- [ ] Rochester, NY
- [ ] San Diego, CA
- [ ] Syracuse, NY
- [ ] Washington, DC

**Comment by jfc3** [comment]
Being that this repository is the same as the [ATE HERE](https://github.com/jfc3/atehere) one only with smaller JSON file sizes to keep the total page load below 10K, I have stopped doing double the work.

If you want to see, the application working live go to [John F Croston III Ate Here](http://jfciii.com/atehere/).

There will be no further code updating at this time unless I try something crazy and don't want to mess with the live version.

Closing out all the open issues too.
